src/collection/collectionplaylistitem.cpp

```
@@ -53,12 +53,12 @@ bool CollectionPlaylistItem::InitFromQuery(const SqlRow &query) {
 
 }
 
-Song CollectionPlaylistItem::Reload() {
+Song CollectionPlaylistItem::Reload(const SharedPtr<TagReaderClient> tagreader_client) {
 
   if (!song_.url().isLocalFile()) return Song();
```

Cppcheck (2.21.0)

[error] There is an unknown macro here somewhere. Configuration is required. If Q_UNUSED is a macro then please configure it.

(unknownMacro)